x-pack/plugins/apm/public/context/url_params_context/types.ts

```diff
@@ -29,4 +29,6 @@ export type IUrlParams = {
   searchTerm?: string;
   percentile?: number;
   latencyAggregationType?: string;
+  comparisonEnabled?: boolean;
+  comparisonType?: string;
```

Comment:
Ideally we could stop adding more urlparams but not sure there are any good alternatives until we tackle https://github.com/elastic/kibana/issues/51963...